Marlin/src/pins/sanguino/pins_MELZI_CREALITY.h

```diff
@@ -38,122 +38,102 @@
 
 #define BOARD_INFO_NAME "Melzi (Creality)"
```

Comment:
MELZI_CREALITY is seeing some pin changes when using CR10_STOCKDISPLAY, which is probably the most common display used with this board.

Old:
```
#define BEEPER_PIN EXP1_01_PIN // (Evaluates to 27)
```

New:
```
#define BEEPER_PIN -1 // Should probably be AUX1_01. This probably was removed because it is frequently repurposed for SERVO0 for BLTouch.
```

Reply:
It looks like the BEEPER change may be related to an incompletely implementation in adapters.h:

```
#elif ENABLED(ADAPTER_CREALITY_PIN27)

  // Creality or BTT "Pin 27" Adapter with BLTouch pins
  ```

Reply:
I fixed `BEEPER_PIN` with this change to `pins_MELZI_CREALITY.h`…
```diff
- #if SERVO0_PIN == BEEPER_PIN || FIL_RUNOUT_PIN == BEEPER_PIN
+ #if PIN_EXISTS(BEEPER) && (SERVO0_PIN == BEEPER_PIN || FIL_RUNOUT_PIN == BEEPER_PIN)
```

Reply:
> … may be related to an incompletely implementation in `adapters.h`:

Not in this case, at least not yet. A "Pin 27" adapter applied to any board's EXP1 connector(s) will make the LCD ignore pin 27 so that a probe or filament sensor can use it. The simplest handling for this would be to add to `adapters.h` above the "set undefined to `-1`" section…
```cpp
#if ENABLED(ADAPTER_PIN27)
  #undef LCD1_01_PIN
#endif
```

We could also look for situations where EXP1_01 is being used for a probe or filament sensor in `adapters.h` or `pins_lcd.h` and alert the user that they need some kind of "Pin 27" adapter. (So users will sometimes need to tell us about their adapter in addition to their LCD after this refactor.)